### DISCUSSION TITLE
Can I change the screenshot for my colorscheme?

### DISCUSSION THREAD

**Discussion by olimorris** [codicocodes/dotfyle#94] [Q&A] [answered]
I'd like for my [colorscheme](https://github.com/olimorris/onedarkpro.nvim) to have a better image on the the homepage.

From this:

![2024-01-08 18_42_11 - Safari@2x](https://github.com/codicocodes/dotfyle/assets/9512444/80cfe1be-30fe-47d4-8263-fbb811828df2)

To this:

![2024-01-08 18_42_40 - Safari@2x](https://github.com/codicocodes/dotfyle/assets/9512444/5c39fc29-a680-47b9-aa6b-eb0089aa8392)

**Comment by codicocodes** [comment, marked as answer]
I've set this to the thumbnail for now (might take some time to propagate through caching). 

It might not stay like that forever though - had to remove the other images as potential thumbnails and in the future I definitely want to have both light mode and dark mode thumbnails.
